### PR TITLE
Added feedstock for grapheme

### PR DESCRIPTION
**Pull request by Brainor**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Brainor** [comment]
@conda-forge/help-python, ready for review!
@conda-forge-admin, please ping conda-forge/help-python

File: recipes/grapheme/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  sha256: 44c2b9f21bbe77cfb05835fec230bd435954275267fea1858013b102f8603cca
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
```
Comment (marcelotrevisani) [inline recipes/grapheme/meta.yaml:19]:
```suggestion
    - python >=3.6
```

File: recipes/grapheme/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/alvinlindstam/grapheme
+  summary: Unicode grapheme helpers
+  license: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (marcelotrevisani) [inline recipes/grapheme/meta.yaml:36]:
```suggestion
  license_file: LICENSE
```

File: recipes/grapheme/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+extra:
+  recipe-maintainers:
+    - Brainor
+    - alvinlindstam
```
Comment (marcelotrevisani) [inline recipes/grapheme/meta.yaml:41]: @alvinlindstam do you want to maintain this recipe as well?

File: recipes/grapheme/meta.yaml (hunk 2 above)
Comment (Brainor) [reply recipes/grapheme/meta.yaml:36]: The [source file](https://github.com/Brainor/staged-recipes/blob/f5b5ed9169b3b576683e02a4b28d2359a79413bb/recipes/grapheme/meta.yaml#L9) `.tar.gz`(the actual [link](https://pypi.io/packages/source/g/grapheme/grapheme-0.6.0.tar.gz)) doesn't contain a license file, so I have to put the license file in `staged-recipes/recipes/grapheme/`[LICENSE](https://github.com/Brainor/staged-recipes/blob/graphmeme/recipes/grapheme/LICENSE), hence the [environ["RECIPE_DIR"]](https://github.com/Brainor/staged-recipes/blob/f5b5ed9169b3b576683e02a4b28d2359a79413bb/recipes/grapheme/meta.yaml#L36) in line 36.

File: recipes/grapheme/meta.yaml (hunk 3 above)
Comment (Brainor) [reply recipes/grapheme/meta.yaml:41]: @alvinlindstam is the author of this package. I guess I can add him as a maintainer when I acquire his permission later.

File: recipes/grapheme/meta.yaml (hunk 2 above)
Comment (Brainor) [reply recipes/grapheme/meta.yaml]: Oh, I see what you mean with this [link](https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually) :)

**Comment by Brainor** [comment]
@conda-forge-admin, please ping conda-forge/help-python

**Comment by Brainor** [comment]
Dear @marcelotrevisani , looks like all problems have been resolved in this PR, could you please check and merge? Thank you. :)

**Review by marcelotrevisani — APPROVED** [review]

**Comment by Brainor** [comment]
Thanks again for your help! 👍

**Pull request merged**